x-pack/plugins/aiops/public/embeddables/change_point_chart/embeddable_chart_component_wrapper.tsx

```diff
@@ -101,10 +101,6 @@ export const ChartGridEmbeddableWrapper: FC<ChangePointDetectionProps> = ({
     10000
   );
 
-  useEffect(() => {
```

Comment:
It was causing an error because onLoading is not defined for the chart change point embeddable component.
![image](https://github.com/user-attachments/assets/f4931a0f-9d01-4f8e-8a2a-4cbbf6e553b8)